docs/docs/reference/other-new-features/export.md

```diff
@@ -87,6 +86,8 @@ def f: c.T = ...
 <a id="note_class"></a>
 Export clauses can appear in classes or they can appear at the top-level. An export clause cannot appear as a statement in a block.
 
+If an export clause contains a wildcard or given selector, it is forbidden for its qualifier path to refers to a package, this is because it is not yet known how to safely track wildcard dependencies to a package for the purposes of incremental compilation.
```

Comment:
Typo "refers" -> "refer". Change the whole thing to: "... path to refer to a package. This is..."